vividus-plugin-applitools/src/main/java/org/vividus/converter/ExamplesTableToRenderingBrowserInfoConverter.java

```diff
@@ -0,0 +1,109 @@
+import com.applitools.eyes.visualgrid.model.IosDeviceInfo;
+import com.applitools.eyes.visualgrid.model.IosDeviceName;
+import com.applitools.eyes.visualgrid.model.IosVersion;
+import com.applitools.eyes.visualgrid.model.ScreenOrientation;
+
+import org.apache.commons.lang3.Validate;
+import org.jbehave.core.model.ExamplesTable;
+import org.jbehave.core.steps.ParameterConverters.AbstractParameterConverter;
+import org.jbehave.core.steps.Parameters;
+
+public class ExamplesTableToRenderingBrowserInfoConverter
+        extends AbstractParameterConverter<ExamplesTable, List<IRenderingBrowserInfo>>
+{
+    @Override
+    public List<IRenderingBrowserInfo> convertValue(ExamplesTable table, Type type)
+    {
+        return table.getRowsAsParameters(true).stream()
+                                              .map(this::parse)
+                                              .collect(Collectors.toList());
+    }
+
+    private IRenderingBrowserInfo parse(Parameters params)
+    {
+        Renderer renderer = params.valueAs("renderer", Renderer.class);
+        return renderer.create(params);
+    }
+
+    private enum Renderer
+    {
+        DESKTOP
+        {
+            @Override
+            IRenderingBrowserInfo create(Parameters params)
+            {
+                return new DesktopBrowserInfo(
+                    RectangleSize.parse(params.valueAs("viewportSize", String.class)),
+                    params.valueAs("browserType", BrowserType.class)
+                );
+            }
+        },
+        IOS
+        {
+            @Override
+            IRenderingBrowserInfo create(Parameters params)
+            {
+                return new IosDeviceInfo(
+                    getDeviceName(params, IosDeviceName::fromName),
+                    getScreenOrientation(params),
+                    params.valueAs("version", IosVersion.class, null)
+                );
+            }
+        },
+        CHROME_MOBILE_EMULATION
+        {
+            @Override
+            IRenderingBrowserInfo create(Parameters params)
+            {
+                return new ChromeEmulationInfo(
+                    getDeviceName(params, DeviceName::fromName),
+                    getScreenOrientation(params)
+                );
+            }
+        };
+
+        abstract IRenderingBrowserInfo create(Parameters params);
+
+        private static <T> T getDeviceName(Parameters params, Function<String, T> converter)
+        {
+            String deviceNameAsString = params.valueAs("deviceName", String.class);
+            T deviceName = converter.apply(deviceNameAsString);
+            Validate.isTrue(deviceName != null, "Unknown device name: %s", deviceNameAsString);
+            return deviceName;
+        }
+
+        private static ScreenOrientation getScreenOrientation(Parameters params)
+        {
+            return params.valueAs("screenOrientation", ScreenOrientation.class, ScreenOrientation.PORTRAIT);
+        }
+    }
```

Comment:
we need better handling if irrelevant parameters/columns are provided for particular renderer